template/web/gatsby-node.js

```diff
@@ -11,9 +11,7 @@ async function createBlogPostPages (graphql, actions) {
   const {createPage} = actions
   const result = await graphql(`
     {
-      allSanityPost(
-        filter: { slug: { current: { ne: null } }, publishedAt: { ne: null } }
-      ) {
+      allSanityPost(filter: {slug: {current: {ne: null}}, publishedAt: {ne: null}}) {
```

Comment:
Might be a prettier or eslint thing(?), but I think the original formatting is more readable, might even add more lines between the different filters

Reply:
This is an example of what running `npm run format` results in. Not sure how to instruct `prettier-eslint` to break lines in such cases. Will fix manually for now.